openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClassesWorkloadTest_Softmx_Increase_JitAot.java

```diff
@@ -233,10 +233,10 @@ public void execute(StfCoreExtension test, StfSharedClassesExtension sharedClass
 				"PS2", ECHO_OFF, ExpectedOutcome.exitValue(1).within("1m"), cacheStatsVM);
 
 		// Compare the AOT and JIT data bytes written to the cache before and after the softmx increase
-		// and make sure the latter is a larger value  
+		// and make sure the latter is a larger or equal value  
 		test.doRunForegroundProcess("Compare AOT and JITdata space before and after softmx was increased. "
```

Comment:
"AOT and JITdata space" should be "AOT and JIT data bytes"

Reply:
Updated.